### PR TITLE
Bump to Kubeless v0.6.0

### PR DESCRIPTION
**Pull request by zanhsieh**
Modify instructions and sample codes to Kubeless v0.6.0.

### CODE REVIEW

File: getting-started/assets/hello.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 module.exports = {
-  handler: (req, res) => {
-    console.log(req.body);
-    res.end(JSON.stringify(req.body));
-  },
-};
+  handler: function (event, context) {
+    return 'hello world!';
```
Comment (andresmgot) [inline getting-started/assets/hello.js]: to maintain the same function this should be:
```
module.exports = {
  handler: (event, context) => {
    console.log(event);
    return event.data;
  },
};
```

**Review by andresmgot — APPROVED** [review]
Apart from my comment the changes LGTM. Thanks for updating this guide!

**Comment by zanhsieh** [comment]
fixed. @andresmgot

**Comment by andresmgot** [comment]
cc/ @sebgoa @zanhsieh they should have write access to this repo

**Pull request merged**